editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -823,6 +823,16 @@ void SpatialEditorViewport::_smouseexit() {
 	_remove_preview();
 }
 
+void SpatialEditorViewport::_sfocusentered() {
+
+	view_menu->show();
```

Comment:
I don't like the idea of hiding, would prefer a function that disables shortcuts on it (on MenuButton), so it can be disabled when not focused.

Reply:
Ok

Reply:
a disable all shortctus mechanism was done, following your suggestions... all changes wwere squashed into one commit, that you can review